client/package.json

```diff
@@ -108,7 +108,7 @@
     "start-production": "NODE_ENV=production node server.js",
     "test": "jest",
     "coveralls": "cat coverage/lcov.info | coveralls",
-    "lint": "eslint app server.js && stylelint --allow-empty-input src && sass-lint -v",
+    "lint": "eslint app server.js && stylelint app/scripts && sass-lint -v",
```

Comment:
I played around with stylelint-scss and other things, to no avail. As somebody not knowledgeable, this takes a bit much time to investigate. Happy to file an issue about this to track it further.

Reply:
I retract my last comment. Apparently we run `sass-lint` over the files in this directory. The current fix should be good. A look over it from Simon or Filip wouldn't hurt though.

Reply:
There is no directory `src` in the same place as `app/scripts`, so this line wasn't checking anything before?

Reply:
Yes. I didn't do any git archaeology to find out if there ever was a `src` directory, but at least this is checking the scripts now and as a I said, `sass-lint` takes care of the rest.